test/unit/views/partials/HeaderSpec.scala

```diff
@@ -59,9 +60,7 @@ class HeaderSpec extends ControllerSpec {
     }
 
     "not be present when a user isn't logged in" in {
-      AuthBuilder.withNotLoggedInUser(mockAuthConnector)
-
-      val result = controller.start().apply(SessionBuilder.buildRequestWithSessionNoUser)
+      val result = controller.start().apply(FakeRequest())
```

Comment:
I think more work may be required to assess / fix the impact of this change to auth-client